lib/base_bot.js

```diff
@@ -548,14 +553,26 @@ class BaseBot extends EventEmitter {
     }
 
     const quickReplies = [];
-    for (const buttonTitle of buttonTitles) {
-      quickReplies.push({
-        content_type: 'text',
-        title: buttonTitle,
-        payload: buttonTitle, // indeed, in default mode payload is buttonTitle
-      });
+
+    // create quick reply option based on bot type
+    // default - else type is facebook messenger
+    if (this.type === "twitter-dm") {
+      for (const buttonTitle of buttonTitles) {
+        quickReplies.push({
+          label: buttonTitle,
+          metadata: buttonTitle,
+        });
+      }
+    } else {
+      for (const buttonTitle of buttonTitles) {
+        quickReplies.push({
+          content_type: 'text',
+          title: buttonTitle,
+          payload: buttonTitle, // indeed, in default mode payload is buttonTitle
+        });
+      }
     }
-    outgoingMessage.addQuickReplies(quickReplies);
+    outgoingMessage.addQuickReplies(this.type, quickReplies);
```

Comment:
`this.type` is also available in the `botmaster-twitter-dm` BotClass object. No need to include it here. `addQuickReplies` should have a different behaviour, this should be done by overriding the method in the `TwitterDMBot` class. But it doesn't look necessary here.